tools/swift-inspect/CMakeLists.txt

```diff
@@ -27,6 +27,20 @@ if(WIN32)
     Sources/SwiftInspectClient/SwiftInspectClient.cpp)
   target_link_libraries(SwiftInspectClient PRIVATE
     SwiftInspectClientInterface)
+elseif(LINUX)
+  string(APPEND CMAKE_Swift_FLAGS " -Xcc -D_GNU_SOURCE")
+  add_library(SwiftInspectLinux STATIC
+    Sources/SwiftInspectLinux/ElfFile.swift
+    Sources/SwiftInspectLinux/LinkMap.swift
+    Sources/SwiftInspectLinux/MemoryMap.swift
+    Sources/SwiftInspectLinux/Process.swift
+    Sources/SwiftInspectLinux/ProcFS.swift
+    Sources/SwiftInspectLinux/SymbolCache.swift)
+  target_link_libraries(SwiftInspectLinux PRIVATE
+    LinuxSystemHeaders)
+  add_library(LinuxSystemHeaders INTERFACE)
```

Comment:
I would have this library precede `SwiftInspectLinux` and add a newline between them.